reference/robot.md

```diff
@@ -126,6 +129,72 @@ void wb_robot_init();
 void wb_robot_cleanup();
 ```
 
+%tab-end
+
+%tab "C++"
+
+```cpp
+#include <webots/Robot.hpp>
+
+namespace webots {
+  class Robot : public Device {
+    Robot();
+    virtual ~Robot();
+    virtual int step(int sampling_period);
```

Comment:
it would be better to use `duration` (as in C API) instead of `sampling period` also for the C++, Java, Python and MATLAB